### ISSUE TITLE
The option '--app-category-type' is not registered with the electron:package command

### ISSUE THREAD

**Issue by oskarrough** [felixrieseberg/ember-electron#40]
Hi, I'm on ember-cli 2.3.0 and ember-electron 1.1.0. When I run `ember electron:package --app-category-type=public.app-category.music` I get the following warning:

```
The option '--app-category-type' is not registered with the electron:package command.
Run ember electron:package --help for a list of supported options.
```

Did I miss something obvious?

**Comment by felixrieseberg** [comment]
No, you found a bug - thanks for reporting. That being said, to fix this *right now*, you can put it into your package.json - that should work. I'll fix the bug later today.

**Comment by felixrieseberg** [comment]
This is now fixed in 1.1.1!
